README.md

```diff
@@ -42,20 +42,13 @@ Building and running with Docker
 Launch images migration from V5 to V6
 -------------------------------------
 
-Migration takes source images from v5 S3 read-only bucket, which could be
-defined by environment variables, example:
+Migration get list of images from v5 database which could be defined by
```

Comment:
"The migration retrieves a list of images from the v5 database. The connection to the database can be defined as environment variable, for example:"